src/components/sync-connected-sites.tsx

```diff
@@ -299,10 +299,10 @@ const SyncConnectedSitesSection = ( {
 														isAnySiteSyncing
 															? isAnyConnectedSiteSyncing
 																? __(
-																		'This Studio site is syncing. Please wait for the sync to finish before you pull this site.'
+																		'This Studio site is syncing. Please wait for the sync to finish before you pull it'
 																  )
 																: __(
-																		'Another Studio site is syncing. Please wait for the sync to finish before you pull this site.'
+																		'Another Studio site is syncing. Please wait for the sync to finish before you pull it.'
```

Comment:
Oh, I probably wasn't clear in my previous suggestion. 😬  I was referring to the strings that start with `This Studio site` only.

I think it still makes sense to keep the following:

```suggestion
																		'Another Studio site is syncing. Please wait for the sync to finish before you pull this site.'
```

Reply:
Hey hey! Actually I think you were clear in your suggestions! I was refactoring some things and applied the changes accidentally. I will implement correct wording now 👌